storage/factory/openStorage.go

```diff
@@ -271,37 +284,85 @@ func getLastEpochAndRoundFromStorage(
 			config.BootstrapStorage.DB.FilePath,
 		)
 
-		bootstrapData, errGet := getBootstrapDataForPersisterPath(persisterFactory, persisterPath, marshalizer)
+		bootstrapData, storer, errGet := getBootstrapDataForPersisterPath(persisterFactory, persisterPath, marshalizer)
 		if errGet != nil {
 			continue
 		}
 
 		if bootstrapData.LastRound > highestRoundInStoredShards {
+			shardID := uint32(0)
+			var err error
+			shardID, err = convertShardIDToUint32(shardIdStr)
```

Comment:
Why not shardID, errConvert := and you can remove both lines above

Reply:
done